### PR TITLE
Conversation access control

### PR DESCRIPTION
**Pull request by seibelsabrina**
Added the new method and other things we specified. How would I compare the owner of the convo with the owner given in the params?  PTAL

### CODE REVIEW

File: src/codeu/chat/client/core/Controller.java [hunk 1]
```diff
@@ -28,9 +28,12 @@
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.Connection;
 import codeu.chat.util.connections.ConnectionSource;
+import java.util.ArrayList;
 
 final class Controller implements BasicController {
 
+  public final ArrayList<Uuid> members = new ArrayList<>();
```
Comment (efuquen) [inline src/codeu/chat/client/core/Controller.java]: This should be a member of the ConversationHeader here:

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/master/src/codeu/chat/common/ConversationHeader.java#L56

You would then end up doing something like `conversation.members.add(member)` in the `addMemberToConvo` function below.

File: src/codeu/chat/client/core/Controller.java [hunk 2]
```diff
@@ -28,9 +28,12 @@
 import codeu.chat.util.Uuid;
```
Comment (efuquen) [inline src/codeu/chat/client/core/Controller.java]: All the below changes should be on the server controller, not client.

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/master/src/codeu/chat/server/Controller.java

File: src/codeu/chat/client/core/Controller.java [hunk 3]
```diff
@@ -39,6 +42,13 @@ public Controller(ConnectionSource source) {
     this.source = source;
   }
 
+  public void addMemberToConvo(Uuid conversationID, Uuid owner, Uuid member) {
+    // look up conversation header based off ID
```
Comment (efuquen) [inline src/codeu/chat/client/core/Controller.java]: In the server controller you can get the conversation by using:

`model.conversationById().first(conversationID)`

File: src/codeu/chat/server/Controller.java [hunk 4]
```diff
@@ -39,6 +41,9 @@ public Controller(Uuid serverId, Model model) {
     this.uuidGenerator = new RandomUuidGenerator(serverId, System.currentTimeMillis());
   }
 
+  public void addMemberToConversation(Uuid conversationID, Uuid owner, Uuid member) {
+    ConversationHeader.members.add(member);
```
Comment (efuquen) [inline src/codeu/chat/server/Controller.java]: You need to get an instance of conversation from the `model`, you are just accessing the class here.

Comment (efuquen) [reply src/codeu/chat/server/Controller.java]: This is an example of how you get a conversation by Id

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/ff639447ffe6d2e2f46e8781a0ba7be10fbc1c18/src/codeu/chat/server/Controller.java#L165

**Comment by seibelsabrina** [comment]
I updated the addMemberToConversation method and it compiles now. How would I compare the conversationID/owner to see if adding the member should be allowed?

**Comment by seibelsabrina** [comment]
PTAL

**Pull request merged**